### PR TITLE
Changelog - 20.07.2, 20.03.6, 1.2.8

### PR DESCRIPTION
**Pull request by parasssh**
---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6688)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
...
 and this project will adhere to [Calendar Versioning](https://calver.org/) starting v20.03.
 
+## [20.07.2] - Unreleased
+
+[20.07.2]: https://github.com/dgraph-io/dgraph/compare/v20.07.1...v20.03.2
+
+### Changed
+
+- Updater badger to 5e3d4b9 (#6669)
+- docs: Update restore documentation. (#6653)
```
Comment (lgalatin) [inline CHANGELOG.md:14]: we normally don't include documentation and testing changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
...
+- doc: update /admin endpoint documentation. (#6415) (#6527)
+- test: Enterprise License tests (#5822) (#6640)
+- build: Makefile to build Dgraph inside docker container. (#6601)
+- docs: add documentation for clone backend (#6622)
+- chore(query): Return content length header for queries (#6469) (#6480)
+- perf(raft): Use raft storage in managedmode (#6457) (#6547)
```
Comment (lgalatin) [inline CHANGELOG.md:20]: only one PR number should be included.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
...
+- feat(Dgraph): add utility to export backup data. (#6590)
+- feat(Dgraph): Add separate compression flag for z and wal dirs (#6401) (#6421)
+
+### Fixed
+
+- fix(Auth): fix Poor-man's auth for admin operations (#6660) (#6686)
+- Break out if g.Ctx is done (#6675)
+- fix(Query): Fix wrong path response for k-shortest paths (#6437) (#6654)
+- fix (graphql): disallowing field names with as (#6645)
+- fix(GraphQL Query): Remove extra fields when querying interfaces (#6596) (#6647)
```
Comment (lgalatin) [inline CHANGELOG.md:36]: please put all GraphQL changes together so we can stay consistent from previous versions.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project will adhere to [Calendar Versioning](https://calver.org/) starting v20.03.
 
+## [20.07.2] - Unreleased
+
```
Comment (lgalatin) [inline CHANGELOG.md:8]: extra line here

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
...
+- docs: add documentation for clone backend (#6622)
+- chore(query): Return content length header for queries (#6469) (#6480)
+- perf(raft): Use raft storage in managedmode (#6457) (#6547)
+- Update index.md (#6566) (#6567)
+- Changes github.com/dgraph-io/graphql-transport-ws version (#6462) (#6529)
+- docs: Update list documentation (#6455) (#6461)
+
+### Added
+
+- feat(Dgraph): add utility to export backup data. (#6590)
```
Comment (lgalatin) [inline CHANGELOG.md:27]: It is better if we can remove this part: feat(Dgraph)

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,126 @@ All notable changes to this project will be documented in this file.
...
+- fix(GraphQL): fix for deletion on interfaces with no non Id field (#6387) (#6417)
+- fix(GraphQL): fix internal error when doing GraphQL schema introspection after drop all (#6268) (#6525)
+- fix(GraphQL): Link xids properly if there are duplicate xids within t… (#6521)
+- fix(GraphQL): Fix query rewriting for auth delete when deleting types with inverse field. (#6350) (#6524)
+- fix(GraphQL): Fix order and offset in auth queries. (#6221) (#6366)
+- Fix(increment): Fix readTs less than minTs (#6317) (#6517)
+- Fix(docs): Fix subsection references in docs (#6439) (#6516)
+- fix(Dgraph): fix bug when deleting and adding to a single UID predicate in the same transaction. (#6449)
+- fix(GraphQL): Generate correct schema when no orderable field in a ty… (#6460)
+- fix(GraphQL): don't generate orderable enum value for list fields (#6392) (#6413)
```
Comment (lgalatin) [inline CHANGELOG.md:63]: ... at the end of sentence, can we fix this.

**Review by lgalatin — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,105 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project will adhere to [Calendar Versioning](https://calver.org/) starting v20.03.
 
+## [20.07.2] - Unreleased
+[20.07.2]: https://github.com/dgraph-io/dgraph/compare/v20.07.1...v20.03.2
```
Comment (lgalatin) [inline CHANGELOG.md:8]: change to: [20.07.2]: https://github.com/dgraph-io/dgraph/compare/v20.07.2...v20.07.1

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,105 @@ All notable changes to this project will be documented in this file.
...
+- Add separate compression flag for z and wal dirs (#6421)
+
+### Fixed
+
+- GraphQL
+  - disallowing field names with as (#6645)
+  - Remove extra fields when querying interfaces (#6647)
+  - fix object Linking with `hasInverse` (#6648)
+  - Update gqlgen in go.mod (#6646)
+  - This PR adds documentation for Scalar DateTime type. (#6554)
```
Comment (lgalatin) [inline CHANGELOG.md:31]: don't need this one since it is documentation

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,105 @@ All notable changes to this project will be documented in this file.
...
+  - fix  restoreStatus query with query variables (#6424)
+  - fix for deletion on interfaces with no non Id field (#6417)
+  - fix internal error when doing GraphQL schema introspection after drop all (#6525)
+  - Link xids properly if there are duplicate xids within t… (#6521)
+  - Fix query rewriting for auth delete when deleting types with inverse field. (#6524)
+  - Fix order and offset in auth queries. (#6366)
+  - Generate correct schema when no orderable field in a type (#6460)
+  - don't generate orderable enum value for list fields (#6413)
+  - fix introspection completion bug (#6389)
+- fix Poor-man's auth for admin operations (#6686)
```
Comment (lgalatin) [inline CHANGELOG.md:45]: if this is ACL related it should go under Enterprise

**Review by lgalatin — COMMENTED** [review]
Some sentences are not starting with capital letter and/or are not ending with a period.

**Review by danielmai — APPROVED** [review]
:lgtm: 

> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6688#-:-MJYI8c16XJ_jlfw6CWA:bg7igkv)* status: all files reviewed, 9 unresolved discussions (waiting on @parasssh)

**Pull request closed without merging**